src/lib.rs

```diff
@@ -147,13 +147,15 @@ impl<'a> FMBuilder<'a> {
         self
     }
 
-    #[doc(hidden)]
+    /// If `yes`, then leading whitespace will be ignored in both pattern and text; otherwise
+    /// leading whitespace must match between the two. Defaults to `true`.
```

Comment:
"match betweeen the two" sounds odd.

How about:
> otherwise leading whitespace must be matched.

Reply:
How about just "must match"?

Reply:
Sorry, as in https://github.com/softdevteam/fm/pull/14/commits/4637022d3629b8021a8e9b599fd668923086eb66.